cli/QuickstepCli.cpp

```diff
@@ -335,6 +347,9 @@ int main(int argc, char* argv[]) {
   std::unique_ptr<SqlParserWrapper> parser_wrapper(new SqlParserWrapper());
   std::chrono::time_point<std::chrono::steady_clock> start, end;
 
+#ifdef QUICKSTEP_ENABLE_GOOGLE_PROFILER
+  unsigned int query_num = 0;
```

Comment:
See the latest version. There's a bool variable declared here instead. We need the declaration to be outside the for loop, so there'll still be a code change here.